projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-admin-page.php

```diff
@@ -75,6 +75,16 @@ public function __construct() {
 	 * @param Jetpack $jetpack object.
 	 */
 	public function on_jetpack_loaded( $jetpack ) {
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended		
+		if ( isset( $_GET['page'] ) ) {
+			$page = sanitize_text_field( wp_unslash( $_GET['page'] ) ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			if ( strpos( $page, 'jetpack/' ) === 0 ) {
+				$section = substr( $page, 8 );
```

Comment:
Why is offset set to `8` here? Or is it just a POC for one page? :)

Reply:
to retrieve what's after `jetpack/`

Reply:
Ah, OK, my bad, I forgot that it's the offset, not the length. This line could use a comment for folks like me :)